cmd/p2p/crawl/crawl.go

```diff
@@ -36,7 +37,7 @@ var (
 var CrawlCmd = &cobra.Command{
 	Use:   "crawl [nodes file]",
 	Short: "Crawl a network on the devp2p layer and generate a nodes JSON file.",
-	Long:  "If no nodes.json file exists, run `echo \"{}\" >> nodes.json` to get started.",
+	Long:  "If no nodes.json file exists, run `echo \"[]\" >> nodes.json` to get started.",
```

Comment:
I found this weird tbh... Shouldn't the tool just create the nodes file if it's empty or doesn't exist?